src/utils/env.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file File for some simple environment-related utilities
+ * @author Box
+ */
+
+/**
+ * Return true if we are currently running in a test or development
+ * environment.
+ *
+ * @return {boolean} true if we are running in a test or dev environment
+ */
+export default function isDevEnvironment() {
```

Comment:
The generic name of the util implies that there may be other items exported from it in the future. Should this be a named export rather than a default export?

Reply:
If I make it a named export, I get this:

```
✖ eslint --fix found some errors. Please fix them and try committing again.

/Users/ehoogerbeets/src/box-ui-elements/src/utils/env.js
13:1  error  Prefer default export  import/prefer-default-export
```

Reply:
Export an object that contains the method:

```
export default {
    isDevEnvironment,
};
```

Reply:
Thoughts on this comment, @ehoogerbeets?